velox/common/memory/HashStringAllocator.h

```diff
@@ -116,12 +117,21 @@ class HashStringAllocator : public StreamArena {
       return begin() + size();
     }
 
-    // Returns Header of the next block or null if at the end of arena.
+    /// Returns the Header of the block that is physically next to this block or
+    /// null if this is the last block of the arena.
     Header* FOLLY_NULLABLE next() {
       auto next = reinterpret_cast<Header*>(end());
       return next->data_ == kArenaEnd ? nullptr : next;
     }
 
+    /// Returns the header of the next block in a multi-part allocation. The
+    /// caller must ensure that isContinued() returns true before calling this
+    /// method.
+    HashStringAllocator::Header* FOLLY_NULLABLE nextContinued() {
```

Comment:
Remove FOLLY_NULLABLE as it is not required any more?

Reply:
It prompts a warning in CLion by default so it's better to keep it